nucypher/blockchain/eth/actors.py

```diff
@@ -932,7 +945,10 @@ def initialize_stake(self,
                                            lock_periods=lock_periods)
 
         # Create stake on-chain
-        receipt = self._deposit(amount=new_stake.value.to_nunits(), lock_periods=new_stake.duration)
+        if not only_lock:
```

Comment:
What is `only_lock` here?

Reply:
I know that name is not good,
so this flag is for switching source of new lock between staker balance and unlocked balance inside escrow

Reply:
Is it like "only use locked tokens"?

Reply:
close, but not exactly
depositing includes transfer token and locking them in one tx
so *only lock* for case when there is no transferring, and only locking of unlocked tokens which belong to staker in `StakingEscrow`